### DISCUSSION TITLE
Not sure how the extension is supposed to work

### DISCUSSION THREAD

**Discussion by ghaithdallaali** [jkohlin/custom-in-out#3] [Q&A]
I'm on MacOS v11.5.1, I managed to install https://github.com/itsmeowForks/spicetify-cli/releases/tag/v2.5.0-patch4 to get around a couple of errors spicetify was throwing, and I am now able to launch Spotify successfully. 

I do see  `skip.mjs` in my list of configured extensions:

```
spicetify-cli/spicetify config extensions skip.mjs 
warning Config "extensions" unchanged: skip.mjs is already in the list.
```

But when Spotify opens, I see no cursors/brackets around the playback bar at the bottom to set the beginning and end of the song. What am I doing wrong? Anything I can do to confirm that I am using the extension properly? 

I do see that the extension did register in the Spotify DevTools under `extensions/skip.mjs`. But right clicking the progress bar does nothing. I have the same problem with loopyloop.js however, so it might be something wrong going on on my end?

Any help in debugging this issue is appreciated.

![image](https://user-images.githubusercontent.com/18090716/129653522-193cf060-de7f-45f0-aac2-f7de2d70524f.png)

**Comment by mexiod** [comment]
Have you found any solution to this problem? I am having the same issue unfortunately
